package.json

```diff
@@ -46,8 +48,10 @@
     "webpack-merge": "^4.2.1"
   },
   "dependencies": {
+    "classnames": "^2.2.6",
     "clean-webpack-plugin": "^2.0.1",
```

Comment:
Webpack plugins are still in the runtime dependencies list, which invokes questions about runtime/dev dependencies understanding. Let's just move them to the dev deps.

Reply:
@loginov-rocks fixed